aad/spring-security/servlet/oauth2/login-authenticate-using-private-key-jwt/src/test/java/com/azure/spring/sample/reactive/servlet/oauth2/login/jwt/azure/activedirectory/AzureActiveDirectoryCertificateSignedAssertionFactoryTest.java

```diff
@@ -24,12 +24,12 @@ public void testPfx() throws AzureActiveDirectoryAssertionException {
     }
 
     // TODO support pem file.
-    @Disabled("Pem file is not supported now.")
-    @Test
+     @Disabled("Pem file is not supported now.")
+     @Test
     public void testPem() throws AzureActiveDirectoryAssertionException {
-        test(new AzureActiveDirectoryCertificateSignedJwtAssertionFactory(
-            "src/test/resources/encrypted-private-key-and-certificate.pem", "myPassword1", TEST_TENANT_ID,
-            TEST_CLIENT_ID));
+//        test(new AzureActiveDirectoryCertificateSignedJwtAssertionFactory(
```

Comment:
comment the `@Disabled` and `@Test` annotation doesn't work, check it [here](https://github.com/fangjian0423/azure-spring-boot-samples/actions/runs/3763007207/jobs/6396183014) . So I comment the code.

Reply:
It's weird, could you please do some more investigation?

Reply:
> It's weird, could you please do some more investigation?

check it [here](https://stackoverflow.com/a/55864867/4239209), seems caused by incompatibilities between `maven-surefire-plugin` and `junit5`.

upgrade `maven-surefire-plugin` to 2.22.0 resolve this problem.